src/LTO.ts

```diff
@@ -56,11 +41,27 @@ export class LTO {
     }
   }
 
+  public generateNewSeed(length): string {
+
+    const random = crypto.generateRandomUint32Array(length);
+    const wordCount = dictionary.length;
+    const phrase = [];
+  
+    for (let i = 0; i < length; i++) {
+      const wordIndex = random[i] % wordCount;
+      phrase.push(dictionary[wordIndex]);
+    }
+  
+    random.set(new Uint8Array(random.length));
```

Comment:
What does this line do?

Reply:
I don't see the point of it, it is not a function i coded, it was already there, i will remove it